### ISSUE TITLE
Istio

### ISSUE THREAD

**Issue by luszczynski** [redhat-helloworld-msa/helloworld-msa#36]
Any plan for Istio in this demo?

**Comment by rafabene** [comment]
Yes. We have plans to update it to replace NetFlixOSS by Istio and make this demo follow the slide "Polyglot Microservices Platform circa 2018": https://docs.google.com/presentation/d/1Xh-xxkuIx1yKfEC1nr_tmJVYd2SnZOZetVNOf-AiKo4/edit#slide=id.g39ba3d9c55_0_4 cc @burrsutter
